infrastructure/code/template/details.html

```diff
@@ -22,6 +22,257 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <style>
+    html {
```

Comment:
Not blocking but I did notice that the scrolling is not as easy as it used to be. I need to use the scrollbar to scroll whereas before I could use my touchpad